shell/src/main/java/tachyon/shell/command/CreateLineageCommand.java

```diff
@@ -96,4 +96,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "createLineage <inputFile1,...> <outputFile1,...> [<cmd_arg1> <cmd_arg2> ...]";
   }
+
+  @Override
+  public String getDescription() {
+    return "To be defined.";
```

Comment:
I guess that there is a lack of information from inputFile1,inputFileN outputFile1,outputFileN cmd_arg1 cmd_arg2 ...